replica/table.cc

```diff
@@ -1262,6 +1262,7 @@ table::seal_snapshot(sstring jsondir, std::vector<snapshot_file_set> file_sets)
             ss << ", ";
         }
         ss << "\"" << rf << "\"";
+        co_await coroutine::maybe_yield();
```

Comment:
unrelated change?

Reply:
it's orthogonal indeed.
I can separate it into a patch of its own.